8/porting_guide_8.rst

```diff
@@ -17,82 +17,91 @@ Ansible 8 is based on Ansible-core 2.15.
 
 We suggest you read this page along with the `Ansible 8 Changelog <https://github.com/ansible-community/ansible-build-data/blob/main/8/CHANGELOG-v8.rst>`_ to understand what updates you may need to make.
 
```

Comment:
Ah, and now we see another result of moving the docs/ folder out of ansible/ansible: antsibull cannot find the ansible-core porting guide anymore and include it in the Ansible porting guide... That's now a part of the ansible-documentation git repo.

Which means it just became a **lot** more important to have that repository tagged for releases as well CC @gotmax23.

Ref: https://github.com/ansible/ansible-documentation/issues/66